rcgen/src/key_pair.rs

```diff
@@ -74,6 +76,8 @@ impl KeyPair {
 	/// Generate a new random key pair for the specified signature algorithm
 	///
 	/// If you're not sure which algorithm to use, [`PKCS_ECDSA_P256_SHA256`] is a good choice.
+	/// If passed an RSA signature algorithm, it depends on the backend whether we return
+	/// a generated key or an error for key generation being unavailable.
```

Comment:
nit: might be helpful to mention which backend supports RSA here.

Reply:
Done